packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/ChatbotFooter/ChatbotMessageBarAttach.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { MessageBar } from '@patternfly/virtual-assistant/dist/dynamic/MessageBar';
+import SourceDetailsMenuItem from '@patternfly/virtual-assistant/dist/dynamic/SourceDetailsMenuItem';
+import { Divider, DropdownGroup, DropdownItem, DropdownList } from '@patternfly/react-core';
+import { BellIcon, CalendarAltIcon, ClipboardIcon, CodeIcon, UploadIcon } from '@patternfly/react-icons';
+
+export const ChatbotMessageBarDefaultAttachExample: React.FunctionComponent = () => {
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
+  const [userFacingMenuItems, setUserFacingMenuItems] = React.useState<React.ReactNode>([]);
+
+  const handleSend = (message) => alert(message);
+
+  const onTextChange = (textValue: string) => {
+    if (textValue === '') {
+      setUserFacingMenuItems(initialMenuItems.concat(uploadMenuItems));
+      return;
+    }
+
+    const newMenuItems = findMatchingElements(initialMenuItems, textValue);
```

Comment:
I think this is the piece we're missing in the demo - it's in the other attachment demo (the full-screen one), which I don't know that we need if we're doing this.

Reply:
Here's the function: https://github.com/patternfly/virtual-assistant/blob/v6/packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/demos/ChatbotAttachmentMenu.tsx#L234